gcc/common/config/riscv/riscv-common.c

```diff
@@ -77,6 +77,26 @@ riscv_parse_arch_string (const char *isa, int *flags, location_t loc)
 	    }
 	}
     }
+  else if (*p == 'e')
+    {
+      p++;
+
+      *flags |= MASK_RVE;
+
+      if (*flags & MASK_64BIT)
+	{
+	  error ("E extension only support for RV32");
```

Comment:
"E" is a base ISA, not an extension; maybe write "RV64E is not a valid base ISA"?